### PR TITLE
Fixes #1073 Use django-copyright to avoid outdated copyright notice 

### LINKED ISSUE #1073: Use django-copyright to avoid outdated copyright notice like Copyright 2010-2013.
[django-copyright](https://github.com/arteria/django-copyright/)
screen shot 2014-12-08 at 23 52 36

### PR DESCRIPTION
**Pull request by philippeowagner**

### CODE REVIEW

File: readthedocs/settings/base.py [hunk 1]
```diff
@@ -282,6 +283,8 @@
 
 GRAVATAR_DEFAULT_IMAGE = 'http://media.readthedocs.org/images/silhouette.png'
 
+COPY_START_YEAR = 20010
```
Comment (ericholscher) [inline readthedocs/settings/base.py]: is this meant to be 2010?

**Comment by ericholscher** [comment]
Thanks! :) Will be happy to merge w/ that one nitpick fixed.

File: readthedocs/settings/base.py (hunk 1 above)
Comment (philippeowagner) [inline readthedocs/settings/base.py]: Whooops. That's a type! Let me fix this ...

**Comment by philippeowagner** [comment]
I updated the PR. Starting now with 2010.

**Comment by ericholscher** [comment]
Thanks! :)

**Pull request merged**